lib/modules/manager/mix/index.ts

```diff
@@ -8,6 +8,9 @@ export { extractPackageFile } from './extract';
 export { updateArtifacts } from './artifacts';
 
 export const defaultConfig = {
+  registryAliases: {
+    hexpm: 'https://hex.pm',
+  },
```

Comment:
why is this needed

Reply:
We need the URL for the hexpm registry, which is the default, so we put it into the default config. This is merged with anything the user has supplied, giving us a complete lookup of the URLs. This has an added benefit that if a company wished to host an internal proxy registry, no updates are needed to their mix files; they have a way to point someplace else.

Reply:
you should not add a default url, the datasource has it already, so only set registry url if it's not the default